tests/test_core.py

```diff
@@ -6951,8 +6952,8 @@ def test_demangle_stacks(self, extra_args):
     self.emcc_args += extra_args
     self.set_setting('DEMANGLE_SUPPORT', 1)
     self.set_setting('ASSERTIONS', 1)
-    # when optimizing function names are not preserved by default.
-    if '-O' in str(self.emcc_args):
+    # unless `-g` is pass function names are not preserved by default.
```

Comment:
`is pass` => `is passed`

but i think the comment is misleading after the change. as the code shows, `--profiling-funcs` is enough, so `-g` is not necessary. how about just `ensure function names are preserved`?